devtools/etdump/tests/buffer_data_sink_test.cpp

```diff
@@ -39,11 +39,13 @@ class BufferDataSinkTest : public ::testing::Test {
   size_t buffer_size_;
   void* buffer_ptr_;
   Span<uint8_t> buffer_;
-  std::unique_ptr<executorch::etdump::BufferDataSink> data_sink_;
 };
 
 TEST_F(BufferDataSinkTest, StorageSizeCheck) {
-  Result<size_t> ret = data_sink_->get_storage_size();
+  Result<BufferDataSink> buffer_data_sink = BufferDataSink::create(buffer_);
+  ASSERT_EQ(buffer_data_sink.error(), Error::Ok);
```

Comment:
It'd be good to put this same ASSERT after all `create` calls in this file

Reply:
Thanks for the suggestion! Making `buffer_data_sink` as a private variable in `BufferDataSinkTest` class, and putting ASSERT after creations, for better structure.